### DISCUSSION TITLE
Recommended database adapter for postgres

### DISCUSSION THREAD

**Discussion by testvinder** [nextauthjs/next-auth#7143] [Help]
Do anyone have recommendations on which database adapter to pick for postgres? I was thinking of using the Typeorm adapter but see that it is called typeorm-legacy-adapter and has no updates for 6 months. Is it no longer being maintained? Is there a better option?

**Comment by jtlapp** [comment]
I'm surprised there is no official vanilla Postgres adapter for those not using ORMs. I did find [authjs-adapter-sql](https://github.com/roelandmoors/authjs-adapter-sql), which works with both Postgres and Mysql and seems well maintained despite only having 3 stars. Haven't tried it.
